### PR TITLE
Advertisement development

### PR DESCRIPTION
**Pull request by juancoelhoo**
Advertisement page created.

### CODE REVIEW

**Review by beviggiano — APPROVED** [review]
Great job, Juan! You did an amazing job! It's just like our prototype.

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const App = () => {
         <>
             <BrowserRouter>
                 <Routes>
-                    <Route path="/" element={<Template/>} />
+                    <Route path="/" element={<AdvertisementScreen/>} />
```
Comment (edu-correia) [inline frontend/src/App.tsx:12]: with my latest change, we can already specify the path ( path="/feed" )

File: frontend/src/components/Menu/Menu.css [hunk 2]
```diff
@@ -0,0 +1,133 @@
+@import url("https://fonts.googleapis.com/css2?family=Luckiest+Guy&family=Quicksand:wght@400;700&display=swap");
```
Comment (edu-correia) [inline frontend/src/components/Menu/Menu.css:1]: We will use this font in the whole project, so we can move this import to global.css so we won't need to add this in every css file

**Review by edu-correia — CHANGES_REQUESTED** [review]
Looking very good man, just 2 points to be aware

**Comment by edu-correia** [comment]
Also, your change has some merge conflicts that you need to resolve before we can merge.
You will need to pull the latest version of the main branch and rebase with your changes, if you have any questions on how to do that, please let me know

**Pull request merged**